receiver/hostmetricsreceiver/README.md

```diff
@@ -129,6 +130,7 @@ The following settings are optional:
 - `mute_process_cgroup_error` (default: false): mute the error encountered when trying to read the cgroup of a process the collector does not have permission to read
 - `mute_process_exe_error` (default: false): mute the error encountered when trying to read the executable path of a process the collector does not have permission to read (Linux only)
 - `mute_process_user_error` (default: false): mute the error encountered when trying to read a uid which doesn't exist on the system, eg. is owned by a user that only exists in a container.
+- `disable_wmi` (default: false) disable the use of Windows Management Interface for fetching process information. Please note that certain metrics cannot be collected without WMI disabled.
```

Comment:
I propose `enable_wmi` (default `true`) or maybe even better `wmi_enabled` to avoid double negation with `disable_wmi: false`.

Reply:
I'm not sure that it is a good idea to tie the data retrieved with the underlying technology to get it. Can't we instead rely on logging proper warning at startup if WMI is not available and use the enable/disable metrics to allow the user to suppress the warnings?